### PR TITLE
model chemistry ccsd(t)/aug-cc-pv(d,t)z to Arkane

### PR DESCRIPTION
**Pull request by davidfarinajr**
- added model chemistry 'ccsd(t)/aug-cc-pv(d,t)z'  with atom energies, bond corrections, and vib scale factor
- added SOC for Cl
- changed Carbon hf to 170.03 from latest ATcT data (https://atct.anl.gov/Thermochemical%20Data/version%201.122d/index.php)

### CODE REVIEW

**Comment by alongd** [comment]
Thanks for this awesome addition, @davidfarinajr ! I have several comments, mainly focused on styling. Generally, I recommend reordering the commits.

- Was the commit `saving changes to checkout cat branch` left here by mistake?
- There's also a "Merge Commit" (`Merge remote-tracking branch 'origin/master' into Chlorine`) that should be removed
- The commit ` changed ccsd(t)/aug-cc-pv(d,t)z vib scale to 0.956 (vib analysis with M062X/aug-cc-pVTZ)`:
  - has a long name, please use a compact header, new line, then description
  - Contains many white space deletions. It is OK and desired to improve white scpace styling, but these large additions will interrupt commit history changes. We recommend adding a separate commit of "minor style changes".
  - The commit message implies that freq scale factors were changed, but it includes Cl SOC, C Hf, and a new model chemistry
  - I see both `'ccsd(t)/aug-cc-pv(d,t)z': 0.956` and `'ccsd(t)/aug-cc-pv(d,t)z': 0.968`. Could you explain why there are two different values here? Arkane will only be able to use one of them, hopefully deterministically, but who knows.
- Please change the commit message ` Merge branch 'master' of https://github.com/ReactionMechanismGenerator/RMG-Py into Chlorine` into something more descriptive of the changes.
- Now I see another commit that deletes the wring factor for 'ccsd(t)/aug-cc-pv(d,t)z' I was referring to above. I don't think it's a good practice for a single PR to add a code block and then remove it in a separate commit.

I kindly ask that you review and stick to our [Contributor Guidelines](https://github.com/ReactionMechanismGenerator/RMG-Py/wiki/RMG-Contributor-Guidelines) and fix this PR before I review its content. Also, you should probably ask someone in your group to show you how to do an interactive git rebase with all it's options.

File: arkane/statmech.py [hunk 1]
```diff
@@ -839,6 +842,10 @@ def applyEnergyCorrections(E0, modelChemistry, atoms, bonds,
                 'O-H':  0.07, 'C-O': 0.25, 'C=O': -0.03, 'O-O': 0.26, 'C-N': -0.20,
                 'C=N': -0.30, 'C#N': -1.33, 'N-O': 1.01, 'N_O': -0.03, 'N=O': -0.26,
                 'N-H':  0.06, 'N-N': -0.23, 'N=N': -0.37, 'N#N': -0.64,}
+        elif modelChemistry == 'ccsd(t)/aug-cc-pv(d,t)z//m06-2x/aug-cc-pvtz'':
```
Comment (rwest) [inline arkane/statmech.py]: this line ends with two quotes

**Comment by alongd** [comment]
@davidfarinajr, what's the status of this PR?

**Comment by davidfarinajr** [comment]
It’s way behind master. It only adds a model chemistry and SOC for Cl to arkane. Instead of merging master into this branch and rebasing, It would be easier to create a new branch off master and make a couple of commits to add SOC and model chemistry.

**Comment by alongd** [comment]
Good idea. Feel free to execute it (then force push to this branch, updating this PR)
Please note that we'll start the Python 3 transition sometime next week, and during this period (~1 month?) we won't be able to accept new PRs (or merge old ones). So if you can pull it off soon that would be great, otherwise this PR would have to sit here a bit more

**Comment by amarkpayne** [comment]
@davidfarinajr I have a PR that will be merged in soon that moves all of this data over to the RMG-database [here](https://github.com/ReactionMechanismGenerator/RMG-database/pull/403). Do you mind either push a commit to that branch or (if after the PR has been merged) opening a PR to the RMG-database with this data? If so I think we can close out this PR

**Comment by davidfarinajr** [comment]
@amarkpayne sounds good!  I'll open PR to the RMG-database

**Pull request closed without merging**